src/Pages/AuthPage/ProfileComponents/LoginBox/LoginBox.jsx

```diff
@@ -0,0 +1,121 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import { useEffect } from "react";
+import { useAuthContext } from "../../../../context/Auth.context";
+import { AuthContainer } from "../AuthContainer/AuthContainer";
+import { SignupBox } from "../SignupBox/SignupBox";
+
+export const LoginBox = () => {
+  const { login, testLogger } = useAuthContext();
+  const [userDetails, setUserDetails] = useState({ email: "", password: "" });
+  const [error, setError] = useState({ state: false, text: "" });
+  const [type, setType] = useState(true);
+
+  const validation = /^(?=.*\d)(?=.*[a-z]).{5,10}$/;
```

Comment:
This is really amazing you are taking care of all the edge cases
Kudos to that man

Reply:
thanks